awslogs/bin.py

```diff
@@ -146,6 +146,12 @@ def add_date_range_arguments(parser, default_start='5m'):
                             dest="query",
                             help="JMESPath query to use in filtering the response data")
 
+    get_parser.add_argument("-pretty",
+                            "--pretty-print",
+                            action='store_true',
+                            dest="pretty_print_enabled",
+                            help="Pretty print the json logs, using a 4 spaces indentation")
```

Comment:
"using 4 spaces indentation" would be grammatically correct

Reply:
Thank you :)